nextbest/src/main/java/no/difi/meldingsutveksling/nextbest/logging/ConversationResourceMarkers.java

```diff
@@ -0,0 +1,22 @@
+package no.difi.meldingsutveksling.nextbest.logging;
+
+import net.logstash.logback.marker.LogstashMarker;
+import net.logstash.logback.marker.Markers;
+import no.difi.meldingsutveksling.nextbest.ConversationResource;
+
+public class ConversationResourceMarkers {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1118)